internal/cephfs/util.go

```diff
@@ -73,6 +73,15 @@ func (cs *ControllerServer) validateCreateVolumeRequest(req *csi.CreateVolumeReq
 		}
 	}
 
+	// Allow readonly access mode for volume with content source
```

Comment:
(nit) Make this a function and move it to validate.go?

Reply:
done